src/index.css

```diff
@@ -15,3 +15,34 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+:root {
+  --black-gradient: linear-gradient(
+    144.39deg,
+    #ffffff -278.56%,
+    #adadad -78.47%,
+    #d1d1d1 91.61%
+  );
+  --card-shadow: 0px 20px 100px -10px rgba(215, 215, 215, 0.1);
+}
+
+* {
+  scroll-behavior: smooth;
+}
+
+.feature-card:hover {
```

Comment:
you can simply add the hover using tailwindcss no need to use pure css for it

Reply:
I agree with ayub, please do the hovering with Tailwind.